New package: GenomicDiversity v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: GenomicDiversity
- Repository: https://github.com/darreni/GenomicDiversity.jl
- Created by: @darreni
- Version: v0.1.0
- Commit: 17a03dd4e24cee455a001690cb1518b81eb8f433
- Reviewed by: @darreni
- Reference: https://github.com/darreni/GenomicDiversity.jl/issues/4
- Description: A Julia package for analysis and plotting of genotypic variation and differentiation across the genomes of a set of individuals.
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359807833f59f8c7a4ebeb3d434377156af7b830ff225108370e47a850734e9db5000ea5a97f9ffd862a61577d562605a475e9c053e94dd0c01ceeabf06367abbecd6716ab520639e4891afc8b220c69c6ebd9e707cb805cd2f9bd833cdb704be1d6eb038dadd872544f48f13d634b451101983e09210c654b217ee4d749879058fb -->

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README. If possible, a full [Documenter](https://documenter.juliadocs.org/stable/)-based is recommended, although that's not necessarily a requirement for the initial registration.

darreni commented:
OK, thanks. I am working on adding the documentation. I had already added docstrings to all the functions. This is my first package so I am learning what is required. I am adding a full worked example in the README, and also trying Documenter. I will submit a new registration request after I get the README in better shape.

JuliaRegistrator committed:
New package: GenomicDiversity v0.1.0

UUID: 594eec9e-a086-4631-8fc7-61104af37930
Repo: https://github.com/darreni/GenomicDiversity.jl.git
Tree: 83dc261c4f4eb10f740ea04253fccbddf063ce5e

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
[noblock]

Much better now!

> I will submit a new registration request after I get the README in better shape.

No need… Just add `[noblock]` to [your comment](https://github.com/JuliaRegistries/General/pull/119520#issuecomment-2480893917)

darreni commented:
Thank you. I have now added [noblock] to my comments. Before your message arrived.though, I had submitted a new registration request and closed the previous one. So now I am unsure if the registration will go through. Could you confirm? And is the 3 day waiting period starting now, or 2 days ago? Thank you.

[noblock]

goerz commented:
> I had submitted a new registration request

Those `@JuliaRegistrator register` comments just updated this existing request, which is the way it should be.

> closed the previous one

You just closed issues in your own repo, which really have very little to do with the actual registration. They're just a vessel for you to issue commands to the `@JuliaRegistrator` bot. You actually don't need issues at all, but you could directly comment on the commits that you would want to submit for registration (that's how I typically do it). And, again: as long as you don't change the version/package name/uuid, it will update an existing request instead of creating a new one.

> So now I am unsure if the registration will go through.

It looks like it's on track.

> And is the 3 day waiting period starting now, or 2 days ago?

Two days ago, since you updated the existing request. If you'd submitted an actual new request (new version), it would have restarted.

So, you should be good to go
